clients/extension/scripts/popup.js

```diff
@@ -176,13 +177,13 @@ async function onGenerateReportButtonClick(background, settings) {
     // Close popup once report is opened in a new tab
     window.close();
   } catch (err) {
-    let message = err.message;
+    let message = err.friendlyMessage || err.message;
```

Comment:
nice this is what we get the protocol timeout method now right? :)

Reply:
Yes, now we get these nice messages:
Waiting for DevTools protocol response has exceeded the allotted time. (Method: Browser.getVersion)